R/utils-fetch.R

```diff
@@ -141,4 +141,58 @@ fetchMinMaxIVs <- function(reportObject, stat){
 fetchFieldVisitReadings <- function(reportObject){
   val <- reportObject$readings
   return(val)
+}
+
+#' Fetch the shift points
+#'
+#' @description Given a report object, will pull the shift points
+#' @param reportObject the full report data
+fetchShiftPoints <- function(reportObject){
+  val <- reportObject[["shiftPoints"]]
```

Comment:
shouldn't the body of these functions be:

```r
ratingShifts <- fetchRatingShifts(reportObject)
val <- ratingShifts[["shiftPoints"]]
```

I think it's misleading to have this named `reportObject` since that refers to the un-parsed list that was converted from JSON and still has everything